### PR TITLE
GP2-2944 DAC_Reflow_01

### PR DESCRIPTION
**Pull request by seijihg**
Page 140 of the DAC Report May.

Fixed overflow with high zoom.

After:
Screenshot 2021-06-17 at 19 12 50

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP-2944
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Reviewing help

- [ ] Explains how to test locally, including how to set up appropriate data
- [x] Includes screenshot(s) - ideally before and after, but at least after

### Housekeeping

- [ ] Added all new environment variables to Vault.
- [ ] Cleaned up old feature flags
- [ ] Upgraded any vulnerable dependencies.
- [ ] Ran the `make manage download_geolocation_data` command
- [ ] I have updated security dependencies
- [ ] Python requirements have been re-compiled.
- [x] Frontend assets have been re-compiled.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/sass/components/_modal-cms.scss [hunk 1]
```diff
@@ -26,3 +26,7 @@
 .ReactModal__Body--open {
```
Comment (bobmeredith) [inline core/sass/components/_modal-cms.scss:26]: As you've changed an scss file, there should be a compiled .css and a .map with it.  Perhaps also a package.lock.
Run `npm run build` to generate these.  Same goes for changes to JavaScript .js or .jsx

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,18 +3,20 @@
 ## Pre-release changes - please put everything in the appropriate category below
 
 ### Enhancements
+
```
Comment (bobmeredith) [inline CHANGELOG.md:6]: As with other files, would be good to revert this reformat of changelog.

**Review by bobmeredith — CHANGES_REQUESTED** [review]
scss needs compiling and there is a reformat of changelog.

**Review by bobmeredith — APPROVED** [review]
I'm not sure why there is a built .js file in here as there are no .js changes so your built js should match exactly the one in GH.
Anyway, you need to rebase off develop, then 'npm build' before you can merge.
in theory, the .js update should go away.
BTW, check that you turn off any 'npm watch' before building as it will overwrite the static resources with dev ones.

**Comment by bobmeredith** [comment]
Something has clearly gone wrong with your pull/merge/build process as there are real code changes in the JS file.
image

**Comment by seijihg** [comment]
> I'm not sure why there is a built .js file in here as there are no .js changes so your built js should match exactly the one in GH.
> Anyway, you need to rebase off develop, then 'npm build' before you can merge.
> in theory, the .js update should go away.
> BTW, check that you turn off any 'npm watch' before building as it will overwrite the static resources with dev ones.

Lemme try rebase and see I had watch on when rebuilding so that might caused the problem.

**Pull request merged**